src/main/java/org/gradle/wrapperupgrade/UpgradeWrapper.java

```diff
@@ -222,8 +229,8 @@ private static Params create(WrapperUpgradeDomainObject upgrade, BuildToolStrate
             var gitCheckoutDir = buildDirectory.getAsFile().get().toPath().resolve(Path.of("git-clones", project));
             var rootProjectDir = gitCheckoutDir.resolve(upgrade.getDir().get());
             var rootProjectDirRelativePath = gitCheckoutDir.relativize(rootProjectDir);
-
-            return new Params(project, repository, baseBranch, prBranch, executionRootDir, gitCheckoutDir, rootProjectDir, rootProjectDirRelativePath, latestBuildToolVersion, gitHub);
+            var gitExtraArgs = upgrade.getOptions().getGitCommitExtraArgs().orElse(Collections.emptyList()).get();
```

Comment:
Please name this variable `gitCommitExtraArgs` in this class so it is clearer what these args are about.